homeassistant/components/xiaomi_miio/const.py

```diff
@@ -135,7 +140,7 @@
     + MODELS_LIGHT_BULB
     + MODELS_LIGHT_MONO
 )
-MODELS_VACUUM = ["roborock.vacuum", "rockrobo.vacuum"]
+MODELS_VACUUM = ["roborock.vacuum", "rockrobo.vacuum", "roborock.vacuum.a15"]
```

Comment:
Seeing that there is also interest in limiting sensors per device etc, it would be better to then list all the models.

A lot of checks also have the `model in MODELS_VACUUM` rather than prefix test.

Reply:
I'll make the needed modifications and sensor registrations appropriately.